### DISCUSSION TITLE
Add a callback to "ListView"?

### DISCUSSION THREAD

**Discussion by Horbin-Magician** [slint-ui/slint#5619] [Q&A]
I want to Implement a list that can infinitely load new items whenever the list is scrolled to the bottom, but neither "ListView" or "ScrollView" have any callback.
Is it necessary to add a callback for them that monitors the change of “viewport-y”?

**Comment by ogoffart** [comment]
You're right we should probably have such a signal. I created an issue for it: https://github.com/slint-ui/slint/issues/5664
